doc/usage/04_build/index.md

```diff
@@ -0,0 +1,136 @@
+# raptor build {#usage_build}
+
+[TOC]
+
+## Main Parameters
+
+### -​-input
+
+<div class="tabbed">
+
+- <b class="tab-title">HIBF</b>
+  A layout file produced with `raptor layout`.
+
+- <b class="tab-title">IBF</b>
+  The input file contains paths to the sequence data. Each line may contain multiple paths (separated by a whitespace).
+
+  ```
+  /absolute/path/to/file1.fasta /absolute/path/to/file2.fasta
+  /absolute/path/to/file3.fa.gz
+  ```
+
+  Preprocessed files (See `raptor prepare`) are also supported.
+
+  Supported file extensions are (possibly followed by bz2, gz, or bgzf):<br>
+    &nbsp;&nbsp;• embl<br>
+    &nbsp;&nbsp;• fasta<br>
+    &nbsp;&nbsp;• fa<br>
+    &nbsp;&nbsp;• fna<br>
+    &nbsp;&nbsp;• ffn<br>
+    &nbsp;&nbsp;• faa<br>
+    &nbsp;&nbsp;• frn<br>
+    &nbsp;&nbsp;• fas<br>
+    &nbsp;&nbsp;• fastq<br>
+    &nbsp;&nbsp;• fq<br>
+    &nbsp;&nbsp;• genbank<br>
+    &nbsp;&nbsp;• gb<br>
+    &nbsp;&nbsp;• gbk<br>
+    &nbsp;&nbsp;• sam
+
+</div>
+
+### -​-output
+The output file name.
+
+### -​-threads
+The number of threads to use. Both IBF and HIBF construction can heavily benefit from parallisation.
+
+### -​-quiet
+By default, runtime and memory statistics are printed at the end.
+
+This flag disables this behaviour.
+
+### -​-kmer
+
+<div class="tabbed">
+
+- <b class="tab-title">HIBF</b>
+  Read from layout file. The read value can be overwritten with this option. However, a warning will be emitted to prevent
```

Comment:
What if I want to build an IBF and supply preprocessed Files? K has to be the same as the one used in preprocessing?

Reply:
Do you mean IBF or HIBF?

Reply:
In general, if you use preprocessed input, you cannot change w/k.
I'm actually not sure what happens with `prepare` -> `layout` -> `build`. 
Not sure what parameters `layout` will use, and where `build` will get its parameters from.
I would hope `build` notices that the layout bins are preprocessed and uses w/k from the minimiser files.